### ISSUE TITLE
PKO: Send new package email can crash app and abort package upload

### ISSUE THREAD

**Issue by cacciatc** [KnightOS/KnightOS#349]
I run into this because I don't have an SMTP server with STARTTTLS locally, but this could happen in the wild too if there was something up with the SMTP server I think.

Might tackle this next week.

**Comment by SirCmpwn** [comment]
Yeah, I have to admit that the email code is currently configured specifically to work with my mail server.
